src/ci/docker/run.sh

```diff
@@ -45,20 +45,20 @@ else
     args="$args --env SCCACHE_DIR=/sccache --volume $HOME/.cache/sccache:/sccache"
 fi
 
-exec docker \
-  run \
```

Comment:
Indeed this is much faster, about 4min30sec total for the `ALLOW_PR` builder. But I'm not sure that disabling the `docker run` entirely is intentional?